jinahub/indexers/dbms/LMDBDBMSIndexer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jinaai/jina:master
+
+# install the third-party requirements
+RUN apt-get update && apt-get install -y python3.7 python3.7-dev python3-pip git
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+RUN python3.7 -m pip install -r requirements.txt
+
+# for testing the image
+RUN python3.7 -m pip install pytest && python3.7 -m pytest -s -v tests/
+
+ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Comment:
change pod to executor, to hide the pod as much as possible

Reply:
This line will change very soon anyhow. I believe it should be `pod`, non?

Reply:
we are trying to hide it from the community since we only selll `Executor`, `Flow` and `Document`

Reply:
I'll clean up all the READMEs, manifests, etc in another PR